package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model
+   href="http://download.ros.org/schema/package_format3.xsd"
+   schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+ <name>geographiclib_cmake_module</name>
+ <version>0.0.0</version>
```

Comment:
```suggestion
 <version>0.0.0</version>
```

In order to be released this will need to be a version > 0.0.0.

Reply:
```suggestion
 <version>0.0.1</version>
```
Start out unstable till we know it works?